### PR TITLE
Miles

### PR DESCRIPTION
**Pull request by milesthedisch**
I made some changes to their style guide. Only small ones. Mostly just got rid of class on elements and used the liquid template to render and hide elements needed. 

i.e Only show Side Navigation on shop pages.

### CODE REVIEW

File: js/components.js [hunk 1]
```diff
@@ -4,14 +4,15 @@ jQuery(function($){
...
 
         $(window).resize(function() {
-            if(body.width() < breakpoint) {
-            }
+            // if(body.width() < breakpoint) {
+            // }
         });
 
         $('.hamburger-menu__icon').click(function() {
+            console.log('clicked')
```
Comment (JordanDDisch) [inline js/components.js]: @milesthedisch remove the console.log

File: assets/styles.css.liquid [hunk 2]
```diff
@@ -260,7 +260,7 @@ hr.divider { margin: 15px 0; background: {{ settings.border_color }}; height: 1p
     text-transform: uppercase;
 }
 /* 2.2 - Grid System */
-.wrapper { width: 996px; margin: 0 auto; padding: 0 15px; background: {{ settings.shop_bg_color_content_area }}; }
+.wrapper { width: 1026px; margin: 0 auto; padding: 0 15px; background: {{ settings.shop_bg_color_content_area }}; }
```
Comment (JordanDDisch) [inline assets/styles.css.liquid:263]: @milesthedisch  You shouldn't be editing their liquid css file. Best to use fugly selectors or remove all together.

Comment (milesthedisch) [inline assets/styles.css.liquid:263]: @JordanDDisc I know. I changed their styles to get it done quicker.

**Comment by milesthedisch** [comment]
Hey I cant find any conflicts in my branch. Everything looks fine in the Hamburger-Nav theme.

**Pull request merged**